test/functional/page_objects/settings_page.js

```diff
@@ -193,17 +193,19 @@ export function SettingsPageProvider({ getService, getPageObjects }) {
 
     async getFieldsTabCount() {
       return retry.try(async () => {
-        const text = await testSubjects.getVisibleText('tab-count-indexedFields');
-        return text.replace(/\((.*)\)/, '$1');
+        const text = await testSubjects.getVisibleText('tab-indexedFields');
+        const index = text.indexOf('(');
+        return text.substring(index).replace(/\((.*)\)/, '$1');
```

Comment:
In `EuiTabbedComponent`, tab name cannot contain HTML elements. So, `tab-count` had to be changed to simple `tab` and we needed to create a substring from the tab name.